spec/StatementIdSpec.php

```diff
@@ -12,8 +12,9 @@
 namespace spec\Xabbuh\XApi\Model;
 
 use PhpSpec\ObjectBehavior;
-use Rhumsaa\Uuid\Uuid;
+use spec\Xabbuh\XApi\Model\Uuid\WrongUuid;
```

Comment:
I would define this class in this file right after the spec class (we do not need it anywhere else).